File: Week_7/problem_2-Memo_and_momo.py
# https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/A

a, b, k = map(int, input().split())

if a % k == 0 and b % k == 0:
    print("Both")
elif a % k == 0 and b % k != 0:
    print("Memo")
elif a % k != 0 and b % k == 0:
    print("Momo")
else:
    print("No One")

File: Week_7/problem_2-Memo_and_momo.c
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/B

#include <stdio.h>
#include <limits.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int main()
{
    long long int a, b, k;
    scanf("%lld %lld %lld", &a, &b, &k);

    if (a % k == 0 && b % k == 0)
        printf("Both");
    else if (a % k == 0 && b % k != 0)
        printf("Memo");
    else if (a % k != 0 && b % k == 0)
        printf("Momo");
    else if (a % k != 0 && b % k != 0)
        printf("No One");

    return 0;
}
